README.md

```diff
@@ -45,13 +45,13 @@ There are many networking solutions available to suit a broad range of use-cases
 		return
 	}
 
-	// Create the sandbox for the containr.
+	// Create the sandbox for the container.
+	// NewSandbox accepts Variadic optional arguments which libnetwork and Drivers can use.
```

Comment:
Sandbox is a libnetwork only concept and hence calling it `Drivers can use` is misleading